### DISCUSSION TITLE
release schedule

### DISCUSSION THREAD

**Discussion by fgregg** [pypa/cibuildwheel#1036] [Q&A]
Very grateful for this project. Is there a scheduled release cadence or is it adhoc? 

There's a couple of fixes on main that I would love to have access to as a release.

**Comment by henryiii** [comment]
The issues with PyPy's troubled 3.7.8 release need to be resolved first, then I think there likely will be a release. Either by rolling it back or by waiting until it's working (probably requires a 3.7.9).

**Comment by joerick** [comment]
Releases are ad-hoc, we have to pick our moment when we have all of our ducks in a row with regards to our upstream dependencies, so a fixed cadence is unlikely as it would probably mean more work to track all of that. That said, which features from main are you particularly interested in @fgregg ?

**Comment by fgregg** [comment]
thanks! the fix i'm waiting on this one (https://github.com/pypa/cibuildwheel/commit/e9ef3e975082f6f57e64eebd798219c5a7bcf3fd)

i'm trying to get an external dependency to adopt cibuildwheels that needs this fix, and i'd like to have my PR reference a specific release

**Reply by joerick** [reply]
Gotcha. It has been a while since the last release. I'll try to get the next one out soon, once we have consistent PyPy versions across our platforms.

**Reply by fgregg** [reply]
thanks so much!

**Reply by henryiii** [reply]
I'm rolling back PyPy to 7.3.7 in #1035. We could do a minor release after that then work on PyPy; the downside is we will nearly immediately (assuming PyPy fixes are fast) want to do a new minor release, since we'll be adding a new target, pypy3.9. Or we could wait and get pypy3.9 in, then release. @mattip, any rough estimate when the issues with PyPy will be resolved? On order a few days, or a few weeks, that sort of thing?

**Comment by mattip** [comment]
Something like 10 days. I would like to see what else is broken before rushing a new release.
